src/components/FavoritesSidebar/styled.js

```diff
@@ -36,20 +36,20 @@ export const FavoritesUl = styled.ul`
     flex-direction: column;
 `
 
-export const FavoritesTitle = styled.h1`
+export const SidebarTitle = styled.h2`
     text-align: center;
     color: ${grey.light};
 `
 
-export const FavoritesLink = styled.a`
+export const SidebarLinks = styled.a`
```

Comment:
this component is a single link, so the name should be singular: `SidebarLink`

Reply:
done